### PR TITLE
tests: ovs interface set mac test

### PR DESCRIPTION
**Pull request by EdDev**
Now it is possible to set the mac address of an ovs-interface.

Depends on #664

### CODE REVIEW

File: tests/integration/testlib/ovslib.py [hunk 1]
```diff
@@ -58,15 +58,18 @@ def add_link_aggregation_port(self, name, slaves, mode=None):
                 OVSBridge.Port.LinkAggregation.MODE
             ] = mode
```
Comment (tyll) [inline tests/integration/testlib/ovslib.py]: The year needs to be updated in the header.

File: tests/integration/testlib/ovslib.py [hunk 2]
```diff
@@ -58,15 +58,18 @@ def add_link_aggregation_port(self, name, slaves, mode=None):
...
 
-    def add_internal_port(self, name, ipv4_state):
+    def add_internal_port(self, name, mac=None, ipv4_state=None):
```
Comment (tyll) [inline tests/integration/testlib/ovslib.py]: Please consider adding a `*, ` before `mac` to enforce the keyword arguments and make future changes easier. This change is changing the signature already.

File: tests/integration/testlib/ovslib.py (hunk 1 above)
Comment (EdDev) [reply tests/integration/testlib/ovslib.py]: Done

File: tests/integration/testlib/ovslib.py (hunk 2 above)
Comment (EdDev) [reply tests/integration/testlib/ovslib.py]: Done

**Pull request merged**